bpf/lib/trace.h

```diff
@@ -25,6 +25,7 @@ enum {
 	TRACE_TO_HOST,
 	TRACE_TO_STACK,
 	TRACE_TO_OVERLAY,
+        TRACE_TO_NETWORK,
```

Comment:
This change will break consumers of the cilium monitor unix domain socket, i.e. older versions of the `cilium monitor` CLI (and Hubble 0.5, though we don't really support it on Cilium 1.8+). 

I don't know enough about our compatibility policy for the Cilium monitor socket, i.e. if we allow such breakage or not. cc @joestringer

Reply:
This enum also can't have a new value inserted into the middle of it. Is there a corresponding bpf.h change?

Reply:
Apart from what @gandro said, one nit - [tabs vs spaces](https://www.youtube.com/watch?v=SsoOG6ZeyUI) - in Cilium tabs won, in the C code too. :) Please adjust your indentation.

Reply:
Thanks for reviewing! PTAL :)